tests/Dfe.PlanTech.Web.E2ETests/cypress/e2e/contentsupport/pages/feedback.cy.js

```diff
@@ -1,6 +1,6 @@
 describe('Feedback banner', () => {
   it("should be visible when tracking consented", () => {
-    cy.visit('content/hello-world',
+    cy.visit('content/digital-technology-asset-register',
```

Comment:
I know we've already put this in to dev (right?), but I'm concerned that the content designers will inevitably change this route in the future. How do we prevent that causing issues?

Reply:
Yeah, its definitely imperfect - I was thinking about creating a command which looks through a recommendation (maybe digital tech registers?) for links and accesses it that way, since clicking the actual links is what we do elsewhere. We'd have to keep tests pretty generic and still leaves the possibility of link removal etc. Otherwise mock content is an option - which means we could test the rich content more specifically